examples/jl-lab-extension-demo-explicit2.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>same here (python 2 imports not needed)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Caltech-IPAC/firefly_client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>